app/models/comment.rb

```diff
@@ -1,6 +1,6 @@
 class Comment < ApplicationRecord
   validates_presence_of :content
-  validates_presence_of :upvotes
 
   belongs_to :commentable, polymorphic: true
+  belongs_to :user 
```

Comment:
This really was the better way to make this relationship. Great job walking us through it and getting us on par.

Reply:
lol sorry if I got a little frustrated I didn't mean it if I had a rude tone. But I am really glad this is working now!!! Just don't look at the routes file